### ISSUE TITLE
1.20

### ISSUE THREAD

**Issue by RocKing1001** [RocKing1001/AllayInABottle#4]
# Update to 1.20

Update to 1.20 has been delayed, but I can probably get this done by next week. I had exams and I really needed to not fail...

I have created a `1.20` branch which will be merged into main once everything is done.

**Comment by piguycs** [comment]
#5 Merged
